.github/workflows/functions-firebase.yaml

```diff
@@ -14,6 +14,18 @@ on:
   - cron:  '0 0 * * 0'
 jobs:
   test:
+    strategy:
+      matrix:
+        # Each package in this list will be tested independently.
+        # New packages must be manually added to this list.
+        package:
+          - 'functions/firebase'
```

Comment:
Should we have a comment that this will be removed once the original sample is deleted?

Reply:
I'll be handling the sample deletion, so I dont think its necessary in this case.